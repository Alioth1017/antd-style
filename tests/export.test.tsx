describe('export', () => {
  it('should work', async () => {
    const all = await import('antd-style');

    expect(Object.keys(all)).toMatchSnapshot();
  });
});
