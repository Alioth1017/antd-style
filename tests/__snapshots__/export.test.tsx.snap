// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`export > should work 1`] = `
[
  "extractStaticStyle",
  "setupStyled",
  "createInstance",
  "createStyles",
  "createGlobalStyle",
  "createStylish",
  "css",
  "cx",
  "keyframes",
  "injectGlobal",
  "styleManager",
  "ThemeProvider",
  "StyleProvider",
  "useTheme",
  "useAntdStylish",
  "useAntdTheme",
  "useAntdToken",
  "useResponsive",
  "useThemeMode",
  "legacyLogicalPropertiesTransformer",
  "px2remTransformer",
]
`;
